### PR TITLE
Add API endpoint for get transfer progress by quote history ID

### PR DESCRIPTION
**Pull request by nstoffel-brightwell**
Documentation update

### CODE REVIEW

File: postman/schemas/transfer.yml [hunk 1]
```diff
@@ -2705,6 +2747,13 @@ components:
           type: array
           items:
             type: string
+    transferProgressResponse:
+      type: object
+      properties:
+        transferId:
+          type: string
+        status:
+          type: string
```
Comment (danchimento) [inline postman/schemas/transfer.yml:2756]: Should status here be an enum with a specific set of values? If not, we will want to include all the possible values in the description section for this endpoint.

File: postman/schemas/transfer.yml [hunk 2]
```diff
@@ -2705,6 +2747,13 @@ components:
           type: array
           items:
             type: string
+    transferProgressResponse:
+      type: object
+      properties:
+        transferId:
```
Comment (danchimento) [inline postman/schemas/transfer.yml:2753]: I'm assuming transfer ID will be null until the transfer is complete? If so this should be described in the description for this property.

File: postman/schemas/transfer.yml [hunk 3]
```diff
@@ -2216,6 +2216,48 @@ paths:
...
+          type: string
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/transferProgressResponse"
+              example:
+                transferRecordStatus: "Created"
```
Comment (danchimento) [inline postman/schemas/transfer.yml:2243]: This property doesn't match the property on the schema object. This one is transferRecordStatus. The schema shows a property named "status". Which one is correct?

Comment (nstoffel-brightwell) [reply postman/schemas/transfer.yml:2243]: transferRecordStatus is right. Fixed the schema

**Review by danchimento — DISMISSED** [review]

File: postman/schemas/transfer.yml (hunk 1 above)
Comment (danchimento) [reply postman/schemas/transfer.yml:2757]: These are interesting statuses....we might want to add a description field here explaining what each status means and what I as a consumer should do about it.

Comment (danchimento) [reply postman/schemas/transfer.yml:2757]: Looks good. If you can include those in the description of the spec, I will add them to the dev docs as well

**Review by danchimento — APPROVED** [review]

**Pull request merged**